Resources/Prototypes/_Goobstation/Entities/Structures/Wallmounts/paintings.yml

```
@@ -0,0 +1,10 @@
+# Mocho requested so yea.
+- type: entity
+  parent: PaintingBase
+  id: PaintingBlunt
+  name: Blunt Painting
+  description: A blunt smoking weird-cat thing with a smug look.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)